src/features/datamodel/useBindingSchema.tsx

```diff
@@ -37,7 +37,7 @@ export function useCurrentDataModelGuid() {
   return getCurrentTaskDataElementId({ application, instance, taskId, layoutSets });
 }
 
-export function useCurrentDataModelUrl() {
+export function useCurrentDataModelUrl(includeRowIds: boolean) {
```

Comment:
Do you ever set includeRowIds to false here? I assumed it was only added for backwards (and schema) compatibility.

Reply:
Currently the backend will just generate new random guids (on every fetch) in that case, so no real need to load with `includeRowId=false`.

Do you think that is a reasonable behaviour for backend, or should I be more restrictive and only generate random guids when the element is unlocked?

Reply:
Naah, that works, but also - as long as we don't need those UUIDs, I don't see why we should fetch them. Making sure our e2e tests work _without_ them when referenced in text resources is a strength in and of itself. It's a lot more difficult to take something away in the future than never providing it in the first place, and if app developers go building on those random UUIDs it will just cause confusion anyway.